packages/template-typescript-minimal/app/pages/home.tsx

```diff
@@ -83,12 +82,21 @@ h1 {
 const Home = ({value}: Props) => {
     const [counter, setCounter] = useState(0)
 
-    useEffect(() => {
-        const interval = setInterval(() => {
-            setCounter(counter + 1)
-        }, 1000)
-        return () => clearInterval(interval)
-    }, [counter, setCounter])
+    const {res: appRes} = useExpress()
+    const query = useQuery(['my-query', counter], async () => {
+        const res = await fetch(`https://api.chucknorris.io/jokes/random?couter=${counter}`)
+        if (appRes) {
+            appRes.status(404)
+        }
+        return await res.json()
+    })
```

Comment:
If I understand this correctly, `useExpress` is a new SDK API that provide access to req and res on the server side.

On the client side, res and req will be undefined, right?

nit - Reading this code block, for a second there I thought we serialize and hydrate `req` and `res` on the client side. What if we name this thing `useServerContext` or something like that (include `server` in the name). So we make this very clear this is only applicable on the server side.

Reply:
Yes that is what is does. I'm open to a name change. Let me run with that and see how it feels